docs/root/configuration/other_features/dlb.rst

```diff
@@ -0,0 +1,42 @@
+.. _config_connection_balance_dlb:
+
+DLB Connection Balancer
+=======================
+
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.dlb.v3alpha.Dlb>`
+
+
+This connection balancer extension provides Envoy with low latency networking by integrating with `Intel DLB <https://networkbuilders.intel.com/solutionslibrary/queue-management-and-load-balancing-on-intel-architecture>`_ through the libdlb library.
+
+The DLB connection balancer is only included in :ref:`contrib images <install_contrib>`
+
+Example configuration
+---------------------
+
+.. code-block:: yaml
```

Comment:
please use a `literalinclude` and separate yaml file for this 

(you may need to exclude archs/platforms i guess)

Reply:
Is there an example for excluding archs/platforms?